src/DssSpell.sol

```diff
@@ -71,234 +60,105 @@ contract SpellAction {
-        ChainlogAbstract(CHANGELOG).setAddress("PIP_YFI", PIP_YFI);
-        ChainlogAbstract(CHANGELOG).setAddress("MCD_JOIN_YFI_A", MCD_JOIN_YFI_A);
-        ChainlogAbstract(CHANGELOG).setAddress("MCD_FLIP_YFI_A", MCD_FLIP_YFI_A);
-
-        // Set the YFI PIP in the Spotter
-        SpotAbstract(MCD_SPOT).file(ilk, "pip", PIP_YFI);
-
-        // Set the YFI-A Flipper in the Cat
-        CatAbstract(MCD_CAT).file(ilk, "flip", MCD_FLIP_YFI_A);
-
-        // Init YFI-A ilk in Vat & Jug
-        VatAbstract(MCD_VAT).init(ilk);
-        JugAbstract(MCD_JUG).init(ilk);
-
-        // Allow YFI-A Join to modify Vat registry
-        VatAbstract(MCD_VAT).rely(MCD_JOIN_YFI_A);
-        // Allow the YFI-A Flipper to reduce the Cat litterbox on deal()
-        CatAbstract(MCD_CAT).rely(MCD_FLIP_YFI_A);
-        // Allow Cat to kick auctions in YFI-A Flipper
-        FlipAbstract(MCD_FLIP_YFI_A).rely(MCD_CAT);
-        // Allow End to yank auctions in YFI-A Flipper
-        FlipAbstract(MCD_FLIP_YFI_A).rely(MCD_END);
-        // Allow FlipperMom to access to the YFI-A Flipper
-        FlipAbstract(MCD_FLIP_YFI_A).rely(FLIPPER_MOM);
-        // Disallow Cat to kick auctions in YFI-A Flipper
-        // !!!!!!!! Only for certain collaterals that do not trigger liquidations like USDC-A)
-        // FlipperMomAbstract(FLIPPER_MOM).deny(MCD_FLIP_YFI_A);
-
-        // Allow OsmMom to access to the YFI Osm
-        // !!!!!!!! Only if PIP_YFI = Osm and hasn't been already relied due a previous deployed ilk
-        OsmAbstract(PIP_YFI).rely(OSM_MOM);
-        // Whitelist Osm to read the Median data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_YFI = Osm, its src is a Median and hasn't been already whitelisted due a previous deployed ilk
-        MedianAbstract(OsmAbstract(PIP_YFI).src()).kiss(PIP_YFI);
-        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_YFI = Osm or PIP_YFI = Median and hasn't been already whitelisted due a previous deployed ilk
-        OsmAbstract(PIP_YFI).kiss(MCD_SPOT);
-        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_YFI = Osm or PIP_YFI = Median and hasn't been already whitelisted due a previous deployed ilk
-        OsmAbstract(PIP_YFI).kiss(MCD_END);
-        // Set YFI Osm in the OsmMom for new ilk
-        // !!!!!!!! Only if PIP_YFI = Osm
-        OsmMomAbstract(OSM_MOM).setOsm(ilk, PIP_YFI);
-
-        // Set the global debt ceiling (end of spell)
-        // VatAbstract(MCD_VAT).file("Line", 1227 * MILLION * RAD);
-        // Set the YFI-A debt ceiling
-        VatAbstract(MCD_VAT).file(ilk, "line", 7 * MILLION * RAD);
-        // Set the YFI-A dust
+        // Set the global debt ceiling
+        VatAbstract(MCD_VAT).file("Line", 1_468_750_000 * RAD);
```

Comment:
Wait...this doesn't look right...will `1_468_750_000` parse in Solidity?

Reply:
Pretty cool right haha

Reply:
![aaf](https://user-images.githubusercontent.com/2374718/99089464-5a4bae80-259b-11eb-9f3a-a67e50d17fd8.png)

Reply:
Amazing.